packages/@react-spectrum/checkbox/test/Checkbox.test.js

```diff
@@ -22,7 +22,6 @@ describe('Checkbox', function () {
 
   afterEach(() => {
     onChangeSpy.mockClear();
-    cleanup();
```

Comment:
why did cleanup get removed in some and replaced with unmount in others?

Reply:
so that depended, essentially cleanup was being used somewhat incorrectly and i think it made use of a bug that was fixed in the upgrade
for the most part cleanup just disappears because it's enforced on a per test level (every `it`), however, we had some cleanups that were being used *inside* of tests in order to *cleanup* the rendered tree. the better way to handle this is to unmount the tree though and not rely on cleanup because that may have other unintended effects